### DISCUSSION TITLE
Why my mouse pointer change everytime it enter a new windows

### DISCUSSION THREAD

**Discussion by flitzcore** [prasanthrangan/hyprdots#1942] [Q&A] [answered]
So without any apps open, my mouse would look very normal. But when I open any tabs and hover my mouse, it looks a bit blurry. And when I download something or if any pop up windows open my mouse pointer suddenly gets bigger when it hover on it. Anyone know why this is happen?

**Comment by kRHYME7** [comment, marked as answer]
This is your Hyprland stuff. Try to see for any open issue in there. https://github.com/hyprwm/Hyprland
This is frequent on monitors that are using fractional scaling or 2 monitor with different scaling. (also high DPI)

**Reply by flitzcore** [reply]
Thanks for your response
